Automattic/wp-calypso
# Proposal: preload notifications for better UX

#### Changes proposed in this Pull Request

* Comparing our UX to Facebook's, it seems that preloading notifications is a nice quick win. IMO, clicking on the bell is a pretty hot path, so it makes sense to preload the notifications asynchronously, they don't slow down rendering the page at all.

### Testing instructions
1. Visit the live link [here](https://calypso.live/?image=registry.a8c.com/calypso/app:build-26255).
2. Allow 3rd party cookies to get logged in.
3. Click on the notifications bell, you should see notifications right away.
4. Optional: compare against wordpress.com.

### Before and After

#### Before
https://user-images.githubusercontent.com/17054134/151999595-043f30a0-dde4-4760-a517-638007f168bd.mov

#### After—but after leaving everything load
https://user-images.githubusercontent.com/17054134/152000002-d90f4b94-f623-443b-a3a2-f48460a3b821.mov
